src/components/gui/gui.css

```diff
@@ -34,7 +34,19 @@
 }
 
 .editor-wrapper {
-    flex-basis: 600px;
+    /*
+        This is carefully balanced--  the minimum width at which the GUI will be displayed is 1024px.
+        At that size, the stage pane is 408px wide, with 8px of padding to each side, for a total of 424px.
+        However, we must also add the border width to the stage pane. All-in-all, the stage pane's final width is
+        424px + ($stage-standard-border-width * 2) (one border per left/right side).
+        1024px - 424px is 600px, so all that's left is to subtract the border width twice.
+
+        @todo This is super fragile, and will break if the amount of padding applied to the stage pane is changed.
```

Comment:
Note that this was always the case (it would always have broken if the stage pane padding was changed)-- 600px just happened to be the "magic number" previously.

Reply:
Wait no I'm wrong one sec